R/cchart.R

```diff
@@ -823,6 +822,15 @@ getPPTSettings <- function(chart.type, args, data)
 
     if (!chart.type %in% c("Pie", "Donut"))
     {
+        # Most of the plotly-based charts in Displayr allow both and axis and a zero line
```

Comment:
Would this mean that the axis line will be missing from the export?

Reply:
If they turned off the zero line, then the axis line will be used instead of the zero line. But if they have both on, then we can only show one line (see page 3 in PPT examples). Here, I've set it to prioritise the zero line because to me it looks more prominent, but we could switch that around?

Reply:
Hmm I think that the zero line is more important. Is there a place to document the limitations?